src/ga4gh/core/identifiers.py

```diff
@@ -58,6 +58,20 @@ class VrsObjectIdentifierIs(IntEnum):
     MISSING = 2
 
 
+class PrevVrsVersion(str, Enum):
+    """Define previous VRS versions that are supported for computing digests and
+    identifiers based on the current VRS model
+    """
+
+    V1_3 = "1.3"
+
+    @classmethod
+    def validate(cls, version):
+        if version is not None and version not in cls.__members__.values():
+            err_msg = f"Expected `PrevVrsVersion`, but got {version}"
+            raise ValueError(err_msg)
```

Comment:
I was going back and forth on moving this to a function rather than classmethod. I'm fine with whichever.